soc/xtensa/intel_adsp/ace/include/intel_ace15_mtpm/adsp_shim.h

```diff
@@ -110,8 +110,8 @@ struct cavs_win {
 #define ADSP_DMWBA_ENABLE   BIT(0)
 #define ADSP_DMWBA_READONLY BIT(1)
 
-#define CAVS_CLKCTL_OSC_SOURCE_MASK  BIT_MASK(2)
-#define CAVS_CLKCTL_OSC_REQUEST_MASK (~BIT_MASK(28))
+#define ADSP_CLKCTL_OSC_SOURCE_MASK  (3<<2)
```

Comment:
Sure about this change?
It used to be a mask  (as macro name says), now it is a single bit
pls double check